content/20.pharmaceuticals.md

```diff
@@ -699,52 +699,88 @@ Additionally, different studies used different dosages, number of doses, and met
+This is predicted to reduce the viral load, mitigate disease, and reduce overall hospitalization. 
+While polyclonal antibodies from convalescent plasma can be used as a treatment for COVID-19, this section focuses on current efforts in developing monoclonal neutralizing antibodies against SARS-CoV-2 (excellent reviews regarding convalescent plasma therapy can be found here [@doi:10/ggqr7s; @doi:10.1001/jama.2020.4940]). 
+Specifically, we focus on monoclonal antibodies that have recently been granted emergency use authorization and discuss the challenges in the successful development of monoclonal neutralizing antibodies.
+
+During the first SARS epidemic in 2002, nAbs were found in SARS-CoV-1-infected patients [@doi:10.1111/j.1469-0691.2004.01009.x; @doi:10.1086/423286]. 
+Several studies following up on these findings identified various S-glycoprotein epitopes as the major targets of nAbs against SARS-CoV-1 [@doi:10.1517/14712590902763755]. 
+Coronaviruses use trimeric spike (S) glycoproteins on their surface to bind to the host cell, allowing for cell entry [@doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.02.058]. 
+Each S glycoprotein protomer is comprised of an S1 domain, also called the RBD, and an S2 domain. 
+The S1 domain binds to the host cell while the S2 domain facilitates the fusion between the viral envelope and host cell membranes [@doi:10.1517/14712590902763755]. 
+The genomic identity between the RBD of SARS-CoV-1 and SARS-CoV-2 is around 74% [@doi:10.1101/2020.02.16.951723]. 
+Due to this high degree of similarity, preexisting antibodies against SARS-CoV-1 were initially considered candidates for neutralizing activity against SARS-CoV-2. 
+While some antibodies developed against the SARS-CoV-1 spike protein showed cross-neutralization activity with SARS-CoV-2 [@doi:10.1016/j.tips.2020.07.004; @doi:10.1038/s41586-020-2349-y], others failed to bind to SARS-CoV-2 spike protein at relevant concentrations [@doi:10.1126/science.abb2507]. 
+Cross-neutralizing activities were dependent on whether the epitope recognized by the antibodies were conserved between SARS-CoV-1 and SARS-CoV-2 [@doi:10.1016/j.tips.2020.07.004].
+
+The first human monoclonal neutralizing antibody specifically against the SARS-CoV-2 S glycoprotein was developed using hybridoma technology [@doi:10.1101/2020.03.11.987958], where antibody-producing B-cells developed by mice are inserted into myeloma cells to produce a hybrid cell line (the hybridoma) that is grown in culture. 
+The 47D11 antibody clone was able to cross-neutralize SARS-CoV-1 and SARS-CoV-2.
+This antibody (now ABVV-47D11) has recently entered clinical trials in collaboration with AbbVie. 
+Since then, an extensive monoclonal neutralizing antibody pipeline has been developed to combat the ongoing pandemic, with over 50 different antibodies in clinical trials [@doi:10.1016/j.bsheal.2021.02.001] and two treatments recently receiving emergency use authorization by the FDA.
```

Comment:
Just curious, this may be a very basic question, does there exist any polyclonal Ab treatment options? Is there a reason that monoclonal Ab are used? My intuition is that you'd have a more robust immune response using polyclonal Ab since they recognize multiple epitopes

Reply:
Polyclonal Ab treatment = convalescent plasma treatment. Convalescent plasma contains polyclonal antibodies that someones immune system generated in response to an infection. Monoclonal antibodies are often derived from these polyclonal antibody-containing plasma specimens. People screen for the most effective clone in neutralizing pseudotyped viruses etc and then start producing the monoclonal antibodies recombinantly. Reasons for monoclonal antibodies are:
Scalability (can make as much as necessary)
Reproducibility (every person has different polyclonal antibodies in their plasma so there is such a large heterogeneity in product quality)
Accessibility (finding people that are willing to donate plasma etc is cumbersome)

Reply:
On the one hand side, you are right, polyclonal antibodies are robust and you have the ability to target multiple epitopes but in reality there are usually a few dominant clones that arise during affinity maturation in the b-cell follicles that do the bulk of the work.

Reply:
I see..thank you for clarifying that. That makes sense!

Reply:
Also, antibodies used for treatment must be humanized, by swapping out as much of the mouse portion as possible. otherwise people mount an immune response against them. 

do you define nAB anywhere in the document?

Reply:
@Jeff-Field that is true when the antibody was raised in mice but most of the COVID antibodies are derived from humans, hence no humanization necessary.